tests/e2e/util.go

```diff
@@ -5647,3 +5646,20 @@ func startCSIPods(ctx context.Context, client clientset.Interface, csiReplicas i
 	isServiceStopped := false
 	return isServiceStopped, err
 }
+
+// waitForStsPodsToBeInRunningState function waits till all the pods comes up
+func waitForStsPodsToBeInRunningState(ctx context.Context, client clientset.Interface, namespace string,
+	statefulset *appsv1.StatefulSet) error {
+	waitErr := wait.Poll(pollTimeoutShort, pollTimeoutShort*20, func() (bool, error) {
+		pods := fss.GetPodList(client, statefulset)
```

Comment:
we should also check if number of pods matches the replica count of the sts

Reply:
After this polling method I am checking number of pods should match the replica count for each statefulSet creation.

Reply:
done, 
Pods ready status check, replica count check - covered in polling method.